development/code-reviews.md

```diff
@@ -55,7 +55,7 @@ author.
 ## After your pull request has been approved...
 
 If your branch is out of date you might need to rebase it. Always rebase against
```

Comment:
Alternative suggestion which avoids naming the branch at all – which might be helpful for when we're working on a feature branch, for example:

> If your branch is out of date you might need to rebase it. Always rebase rather than merging the [target / base?] branch into your branch (unfortunately this is what GitHub’s ‘update branch’ button does)

Reply:
Good idea, I prefer that 👍